manifest.yml

```diff
@@ -0,0 +1,18 @@
+---
+# This manifest deploys a Python Flask application with a Cloudant database
+applications:
+- name: nyu-wishlist-service-f20
+  path: .
+  instances: 2
+  memory: 64M
+  routes:
+  - route: nyu-wishlist-service-f20.us-south.cf.appdomain.cloud
+  disk_quota: 1024M
+  buildpacks: 
+  - python_buildpack
+  timeout: 180
+  services:
+  - Cloudant
```

Comment:
We will be using postgres, I think we'll need to do to elephant sql for that. Thoughts?

Reply:
![image](https://user-images.githubusercontent.com/60806958/97785570-2bd1da80-1b7c-11eb-852a-dc9caf8b60f0.png)
Both ok.

Reply:
Ya, but we have postgres everywhere in our application, so cloudant won't work

Reply:
Change to elephantsql now.